grpc-testtool/README.md

```diff
@@ -19,3 +19,23 @@ There are 3 RPC specs that must be implemented:
 2. The sync proto, which supports retrieving range and change proofs
 3. The process-server proto, which currently only retrieves metrics
 
+# Running
+
+To run a single test and make sure things are working, first check out and build the go and rust code.
+See the corresponding README for specific build requirements.
+
+ - BASE=$HOME
+ - cd $BASE && git clone git@github.com:ava-labs/merkledb-tester.git
+ - cd $BASE && git clone git@github.com:ava-labs/firewood.git
+
+Then, build the rust process server and symlink it to where the testtool expects it:
+
+ - cd $BASE/firewood
+ - `cargo build --release`
+ - `ln -sf $BASE/firewood/target/release/process-server $BASE/merkledb-tester/process/process-server`
+
+Then, run the test you want:
+
+ - `cd $BASE/merkledb-tester`
+ - `go test -timeout 2m -run TestConsistency github.com/ava-labs/merkledb-tester/tests/consistency -v`
+ - `go test -timeout 2m -run TestPutPerformance github.com/ava-labs/merkledb-tester/tests/performance -v`
```

Comment:
````suggestion
```sh
cd $BASE/merkledb-tester
go test -timeout 2m -run TestConsistency github.com/ava-labs/merkledb-tester/tests/consistency -v
go test -timeout 2m -run TestPutPerformance github.com/ava-labs/merkledb-tester/tests/performance -v
```
````
You should also mention that you need `go` before you get here

Reply:
The fact that you need go (and rust and other stuff) is mentioned in "See the corresponding README for specific build requirements."

This is mostly tailored for internal users and they should already have those things installed, but if not, they can check out the corresponding README for the repo they are unfamiliar with.